vpp_client/vpp_client_interfaces.go

```diff
@@ -166,6 +166,9 @@ func (v *VppInterface) SearchInterfaceWithName(name string) (err error, swIfInde
 	v.lock.Lock()
 	defer v.lock.Unlock()
 
+	swIfIndex = ^uint32(0)
```

Comment:
Maybe might be worth defining a `vpp_interface.INVALID_SWIFINDEX` in the api wrapper nope ?

Reply:
Good point, will do.